src/components/JehMaker.vue

```diff
@@ -373,6 +381,7 @@ export default class JehMaker extends Vue {
     let urlSplit = this.urlImport.split('/')
     if (urlSplit.length > 2 && urlSplit[urlSplit.length - 2] === 'p') {
       this.importFromB64(urlSplit[urlSplit.length - 1])
+      this.calculate()
```

Comment:
Hello, ça m'a l'air bon.
J'ai juste cette ligne que j'aurais directement intégré dans la méthode loadProject.
A voir

Reply:
je comprends pas trop... ici c'est lors d'un import par l'url, donc pas de lien avec loadProject je pense

Reply:
Si regarde la fonction importb64 c'est elle qui ensuite déclenche loadProject
https://github.com/idesys-dev/jeh-maker/blob/cd93b78763ec9325e852cad6ccb573538801a677/src/components/JehMaker.vue#L375-L378

Reply:
Done :)
ready to merge, ok for you?